Source/src/core/GameObject.cpp

```diff
@@ -65,11 +67,11 @@ Hachiko::GameObject::~GameObject()
     for (GameObject* child : children)
     {
         child->parent = nullptr;
-        RELEASE(child);
+        RELEASE(child)
     }
     for (Component* component : components)
     {
-        RELEASE(component);
```

Comment:
I preffer having a visible ; 😢

Reply:
What is the state of this?

Reply:
we can leave the comma, just note that it's redundant